ivadomed/main.py

```diff
@@ -741,7 +741,7 @@ def cmd_test(context):
             rdict['gt'] = preds.cpu()
             batch.update(rdict)
 
-            if batch["input"].shape[1] > 1:
+            if batch["input"].shape[1] > 1 and not i_monteCarlo:
```

Comment:
Not sure to understand here: why the first iteration of MC is not taking only second channel?

Reply:
Because it is the same batch across the for loop and the operation needs to be done only once :)

Reply:
oooo! makes sense!